src/test/java/org/jabref/gui/autocompleter/FieldValueSuggestionProviderTest.java

```diff
@@ -56,6 +61,23 @@ void completeAfterAddingEntryWithoutFieldReturnsNothing() {
         assertEquals(Collections.emptyList(), result);
     }
 
+    @Test
+    void completeOnIgnoredFieldReturnsNothing() {
+        AutoCompletePreferences autoCompletePreferences = mock(AutoCompletePreferences.class);
+        JournalAbbreviationRepository journalAbbreviationRepository = mock(JournalAbbreviationRepository.class);
+        when(autoCompletePreferences.getCompleteFields()).thenReturn(new HashSet<>(Set.of(StandardField.AUTHOR)));
```

Comment:
```suggestion
        when(autoCompletePreferences.getCompleteFields()).thenReturn(Set.of(StandardField.AUTHOR));
```
should be sufficient, because Set.of already returns a Set